vignettes/BCB420.2019.EnrichDepletTF.Rmd

```diff
@@ -0,0 +1,145 @@
+---
+title: "EnrichDepletTF: querying for enrichement or depletion of TF-binding genes"
+author:
+- name: Sapir Labes
+  affiliation: 
+  - http://orcid.org/0000-0002-9478-5974
+  - University of Toronto
+  - The Hebrew University of Jerusalem
+  email: sapir.labes@mail.huji.ac.il
+date: "`r Sys.Date()`"
+abstract: >
+  This Vignette demonstrates how to query enrichment and depletion of TF-binding genes among significantly correlated pairs of genes of a chosen system.
+output:
+  BiocStyle::html_document:
+    toc: true
+    toc_float: true
+    fig_caption: yes
+vignette: >
+  %\VignetteIndexEntry{BCB420.2019.ESA Vignette}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+references:
+- id: steipe2019
+  title: 'rptPlus: R Package Template - enhanced'
+  author:
+  - family: Steipe
+    given: Boris
+    orcid: 0000-0002-1134-6758
+  container-title: '[R package](https://github.com/hyginn/BCB420.2019.ESA)'
+  URL: 'https://github.com/hyginn/BCB420.2019.ESA'
+---
+
+<!--- this inline style supports diff-like highlighting -->
+<style>
+  .add {
+    color: #22863a;
+    background-color: #f0fff4;
+  }
+  .del {
+    color: #b31d28;
+    background-color: #ffeef0;
+  }
+</style>
+
+
+```{r setup, include = FALSE}
```

Comment:
Remove chunks and text from the template stub that are not relevant for your package.